components/python/requests/Makefile

```diff
@@ -0,0 +1,58 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright (c) 2013, 2016, Oracle and/or its affiliates. All rights reserved.
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		requests
+COMPONENT_VERSION=	2.11.1
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_HASH=	\
+    sha256:e8e5710ae923a00e45631f0e47c30ebfc83b7a8c4a7a818f78bf820f9b607558
+COMPONENT_ARCHIVE_URL=  https://github.com/kennethreitz/requests/archive/v$(COMPONENT_VERSION).tar.gz	
+COMPONENT_PROJECT_URL=	http://python-requests.org/
+COMPONENT_SUMMARY= Python HTTP Requests for Humans
+COMPONENT_CLASSIFICATION= Development/Python
+COMPONENT_LICENSE= Apache 2
+
+PYTHON_VERSIONS= 2.7 3.4
+
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/setup.py.mk
+include $(WS_MAKE_RULES)/ips.mk
+
+ASLR_MODE = $(ASLR_NOT_APPLICABLE)
+
+
+
+# common targets
+build:		$(BUILD_NO_ARCH)
+
+install:	$(INSTALL_NO_ARCH)
+
+# Tests require access to AWS services & the Internet
+test:		$(NO_TESTS)
+
+system-test:	$(NO_TESTS)
```

Comment:
Also we usually don't have system-test target in components (however, there are some exceptions) as we miss SYSTEM_TEST rules.